--- bundle/ngx_stream_lua-0.0.9/src/ngx_stream_lua_ssl_certby.c       2020-10-30 00:38:11.000000000 -0300
+++ bundle/ngx_stream_lua-0.0.9/src/ngx_stream_lua_ssl_certby.c       2020-12-20 15:54:32.656319271 -0300
@@ -1385,0 +1386,7 @@ ngx_stream_lua_ffi_ssl_verify_client(ngx
+#ifdef LIBRESSL_VERSION_NUMBER
+
+    *err = "LibreSSL not supported";
+    return NGX_ERROR;
+
+#else
+
@@ -1489,0 +1497 @@ failed:
+#endif
--- bundle/ngx_lua-0.10.19/src/ngx_http_lua_ssl_certby.c      2020-11-03 03:58:18.000000000 -0300
+++ bundle/ngx_lua-0.10.19/src/ngx_http_lua_ssl_certby.c      2020-12-20 16:00:21.481504383 -0300
@@ -1366,0 +1367,7 @@ ngx_http_lua_ffi_ssl_verify_client(ngx_h
+#ifdef LIBRESSL_VERSION_NUMBER
+
+    *err = "LibreSSL not supported";
+    return NGX_ERROR;
+
+#else
+
@@ -1468,0 +1476 @@ failed:
+#endif
